.github/workflows/e2e-main.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       max-parallel: 2
       fail-fast: false
       matrix:
-        ios-version: ['14.5', '15.0']
```

Comment:
Can we add iOS 17 then? Since we want to test the lowest and highest supported.

Reply:
This needs to be done in a follow up PR. Once we have product approval we can setup the new runners, migrate off the old ones and then enable testing of iOS 17.